echo.py

```diff
@@ -2,21 +2,53 @@
 # -*- coding: utf-8 -*-
 """An enhanced version of the 'echo' cmd line utility"""
 
-__author__ = "???"
+__author__ = "knmarvel"
 
 
-import sys
+import argparse
 
 
 def create_parser():
-    """Creates and returns an argparse cmd line option parser"""
-    pass
+    parser = argparse.ArgumentParser(
+        description="Perform transformation on input text.")
+    parser.add_argument("text",
+                        type=str,
+                        help="text to be manipulated")
+    parser.add_argument('-u',
+                        '--upper',
+                        action='store_const',
+                        const=bool,
+                        help="convert text to uppercase")
+    parser.add_argument("-l",
+                        "--lower",
+                        action='store_const',
+                        const=bool,
+                        help="convert text to lowercase")
+    parser.add_argument("-t",
+                        "--title",
+                        action='store_const',
+                        const=bool,
+                        help="convert text to titlecase")
+    return parser.parse_args()
+
+
+def main():
+    """Implementation of echo"""
 
+    args = create_parser()
+    text = args.text
 
-def main(args):
-    """Implementation of echo"""
-    pass
+    if args.upper:
+        text = text.upper()
+
+    if args.lower:
+        text = text.lower()
+
+    if args.title:
+        text = text.title()
+
+    print(text)
```

Comment:
it looks like you'd be overwriting your variable text several times if someone typed in `$python echo.py "hEllo" -ult `.
try printing within each if-statement here instead.

Reply:
Only one word should print out. I want to overwrite my variables in the order of the argparse (Upper, lower, title) to print out the word in the correct case.